source/test/F0.Minesweeper.Components.Tests/Pages/Game/Modules/CellTests.razor

```diff
@@ -302,12 +256,15 @@
 	public void OnClick_HasAdjacentMinesToUncover_UncoversAllCells()
 	{
 		// Arrange
-		RenderFragment expectedMarkup = @<div><button class="f0-cell f0-cell-uncovered">0</button></div>;
-		Location expectedLocationClickedCell = new(1, 1);
+        RenderFragment expectedMarkup = 
+    @<div><button class="f0-cell f0-cell-uncovered"> </button></div>
+    ;
```

Comment:
Intentional spacing/line-breaking?

Reply:
I don't know why that's formatted in that way by the IDE so I sadly don't know how to fix that permanently on my end. So for now I put it manually back where it belongs.